editor/test/integration/sprite_test.clj

```diff
@@ -21,17 +21,25 @@
 
 (deftest replacing-sprite-image-replaces-dep-build-targets
   (test-util/with-loaded-project
-    (let [node-id (project/get-resource-node project "/logic/session/pow.sprite")
-          old-texture-binding-info (first (g/node-value node-id :texture-binding-infos))
+    (let [sprite-id (project/get-resource-node project "/logic/session/pow.sprite")
+          old-texture-binding-info (first (g/node-value sprite-id :texture-binding-infos))
           old-texture-binding-node-id (:_node-id old-texture-binding-info)
-          old-image (:texture old-texture-binding-info)]
-      (let [old-sources (g/sources-of old-texture-binding-node-id :build-targets)]
```

Comment:
The old test impl relied on explicit build target connections, but now we use dynamic deps there.